src/components/expression/expression-panel.js

```diff
@@ -224,16 +218,66 @@ genotet.ExpressionPanel.prototype.updateGenes = function(gene) {
       text: gene
     };
   });
-
   var profile = this.container.find('#profile select').empty();
-  this.selectProfiles_ = profile.select2({
-    data: genes,
-    multiple: true
-  });
-  this.container.find('#profile .select2-container').css({
-    width: '100%'
-  });
-  this.updateGeneSelections();
+
+  if (!genes.length) {
+    return;
+  }
+
+  $.fn.select2.amd.require(['select2/data/array', 'select2/utils'],
```

Comment:
IIRC when you type-in any character you can only list those items matching the characters. That's equivalent to a search box.

Has the above issue been resolved?